SW/Boiling Detection/boil_detect.py

```diff
@@ -0,0 +1,240 @@
+from skimage.measure import compare_ssim
+import cv2
+import numpy as np
+import time
+
+
+class BoilDetector:
+
+    def __init__self(self):
+        self.temp = 170 # temp considered to be boiling
```

Comment:
We need to verify this with actual experimentation, but it will probably be way less than this.